shared/src/main/scala/uk/gov/homeoffice/drt/feedback/UserFeedback.scala

```diff
@@ -0,0 +1,14 @@
+package uk.gov.homeoffice.drt.feedback
+
+case class UserFeedback(email: String,
+                        actionedAt: Long,
```

Comment:
I'd maybe rename `actionedAt` to `createdAt` since actioned suggested some action has been taken given the feedback